encrypt terraform state (#24)
status-im/infra-template: ansible/README.md

ansible all -o -m debug -a 'var=ansible_host' | columns -t
```

> A backup of the terraform inventory state exists at : `.terraform/terraform.tfstate.backup` This backup is encrypted with a key generated from hashing `CONSUL_HTTP_TOKEN`
> 
> This key is then used to encrypt the terraform state backup with a Fernet Encryption.
> 
> This encrypted backup can be decrypted by using a helper script at in infra-utils repo at : [decrypt_tf_backup.py](https://github.com/status-im/infra-utils/blob/master/terraform/decrypt_tf_backup.py)

Reviewer: Less is more, and not every sentence needs its own paragraph.
```suggestion
A backup of the Terraform state is created at `.terraform/terraform.tfstate.backup`. It is symetrically encrypted using [Fernet algorithm](https://cryptography.io/en/latest/fernet/) with a key generated from haed `CONSUL_HTTP_TOKEN` and can be decrypted by using [`decrypt_tf_backup.py`](https://github.com/status-im/infra-utils/blob/master/terraform/decrypt_tf_backup.py) script.
```
